packages/api/src/util/ensemble.js

```diff
@@ -111,27 +133,27 @@ async function registerOrganizationWebhook(farmId, organizationId, accessToken)
     .where({ farm_id: farmId, partner_id: 1 })
     .first();
   if (existingIntegration?.webhook_id) {
-    return existingIntegration.webhook_id;
-  } else {
-    const axiosObject = {
-      method: 'post',
-      url: `${ensembleAPI}/organizations/${organizationId}/webhooks/`,
-      data: {
-        url: `${baseUrl}/sensor/reading/partner/1/farm/${farmId}`,
-        authorization_header: authHeader,
-        frequency: 15,
-      },
-    };
-    const onError = (error) => {
-      console.log(error);
-      throw new Error('Failed to register webhook with ESCI');
-    };
-    const onResponse = async (response) => {
-      await FarmExternalIntegrationsModel.updateWebhookId(farmId, response.data.id);
-      return { ...response.data, status: response.status };
-    };
-    return await ensembleAPICall(accessToken, axiosObject, onError, onResponse);
+    return;
   }
+
+  const axiosObject = {
+    method: 'post',
+    url: `${ensembleAPI}/organizations/${organizationId}/webhooks/`,
+    data: {
+      url: `${baseUrl}/sensor/reading/partner/1/farm/${farmId}`,
+      authorization_header: authHeader,
+      frequency: 15,
+    },
+  };
+  const onError = (error) => {
+    console.log(error);
+    throw new Error('Failed to register webhook with ESCI');
+  };
+  const onResponse = async (response) => {
+    await FarmExternalIntegrationsModel.updateWebhookId(farmId, response.data.id);
+    return { ...response.data, status: response.status };
+  };
+  await ensembleAPICall(accessToken, axiosObject, onError, onResponse);
```

Comment:
While unused the return statement gave the option to see result of webhook call.